### PR TITLE
azurerm_mssql_elasticpool: moved `elastic_pool_properties` properties to the top level

### PR DESCRIPTION
**Pull request by katbyte**
Also fixes some potential crash points.

### CODE REVIEW

File: azurerm/resource_arm_mssql_elasticpool.go [hunk 1]
```diff
@@ -119,39 +120,70 @@ func resourceArmMsSqlElasticPool() *schema.Resource {
...
 
+			"state": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"creation_date": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```
Comment (tombuildsstuff) [inline azurerm/resource_arm_mssql_elasticpool.go]: I think we’d be better to remove the state and creation date fields entirely here?

File: azurerm/resource_arm_mssql_elasticpool.go [hunk 2]
```diff
@@ -119,39 +120,70 @@ func resourceArmMsSqlElasticPool() *schema.Resource {
...
+
+			"zone_redundant": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+
+			"license_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```
Comment (tombuildsstuff) [inline azurerm/resource_arm_mssql_elasticpool.go]: I can’t think of when this field would be likely to be used from within Terraform - should we remove it?

File: azurerm/resource_arm_mssql_elasticpool.go [hunk 3]
```diff
@@ -119,39 +120,70 @@ func resourceArmMsSqlElasticPool() *schema.Resource {
...
 						"license_type": {
-							Type:     schema.TypeString,
-							Computed: true,
+							Type:       schema.TypeString,
+							Computed:   true,
+							Deprecated: "This property has been moved to the top level",
 						},
 					},
 				},
 			},
```
Comment (tombuildsstuff) [inline azurerm/resource_arm_mssql_elasticpool.go]: Can we add docs for these new fields?

File: azurerm/resource_arm_mssql_elasticpool.go (hunk 1 above)
Comment (katbyte) [reply azurerm/resource_arm_mssql_elasticpool.go]: Good call

Comment (katbyte) [reply azurerm/resource_arm_mssql_elasticpool.go]: I can see them potentially being useful, but i do agree those would be edge cases.

**Comment by katbyte** [comment]
tests pass:
screen shot 2018-11-22 at 15 21 57
